### PR TITLE
elantp: Use fu_device_retry_full() to avoid a Coverity warning

### PR DESCRIPTION
**Pull request by hughsie**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [X] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

**Comment by jinglewu** [comment]
It is retry 3 times for only current page, not all page retry again.

File: plugins/elantp/fu-elantp-hid-haptic-device.c [hunk 1]
```diff
@@ -731,14 +695,65 @@ fu_elantp_hid_haptic_device_write_firmware(FuDevice *device,
...
+	/* write each block */
+	helper_write.fw = fw;
+	helper_write.progress = fu_progress_get_child(progress);
+	if (!fu_device_retry_full(device,
+				  fu_elantp_hid_haptic_device_write_chunks_cb,
+				  3,
+				  100,
+				  &helper_write,
+				  error))
+		return FALSE;
```
Comment (jinglewu) [inline plugins/elantp/fu-elantp-hid-haptic-device.c:756]: It is retry 3 times for only current page, not all page retry again.

Comment (hughsie) [reply plugins/elantp/fu-elantp-hid-haptic-device.c:757]: Understood; can you try the fixed version please.

Comment (jinglewu) [reply plugins/elantp/fu-elantp-hid-haptic-device.c:757]: It's works, but some tp firmware have "tp busy issue, when write the page, ", it's hard to reproduce.

**Pull request merged**